### PR TITLE
[github] Updated deprecated workflows actions

### PR DESCRIPTION
**Pull request by phunkyfish**
Updated deprecated workflows actions versions., Merge once green

### CODE REVIEW

File: .github/workflows/increment-version.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+on:
+  push:
+    branches: [ Matrix, Nexus ]
+    paths:
+      - '**resource.language.**strings.po'
+
+jobs:
+  default:
+    if: github.repository == 'kodi-pvr/pvr.iptvsimple'
```
Comment (Rechi) [inline .github/workflows/increment-version.yml]:
```suggestion
    if: github.repository == 'xbmc/inputstream.rtmp'
```

File: .github/workflows/sync-addon-metadata-translations.yml [hunk 2]
```diff
@@ -9,25 +9,37 @@ on:
 
 jobs:
   default:
-    if: github.repository == 'xbmc/inputstream.rtmp'
+    if: github.repository == 'kodi-pvr/pvr.iptvsimple'
```
Comment (Rechi) [inline .github/workflows/sync-addon-metadata-translations.yml]:
```suggestion
    if: github.repository == 'xbmc/inputstream.rtmp'
```

File: .github/workflows/increment-version.yml (hunk 1 above)
Comment (phunkyfish) [reply .github/workflows/increment-version.yml]: Fixed, thanks

File: .github/workflows/sync-addon-metadata-translations.yml (hunk 2 above)
Comment (phunkyfish) [reply .github/workflows/sync-addon-metadata-translations.yml]: Fixed, thanks

**Pull request merged**